packages/salesforcedx-vscode-core/src/commands/util/postconditionCheckers.ts

```diff
@@ -221,39 +226,72 @@ export class ConflictDetectionChecker implements PostconditionChecker<string> {
       };
       const results = await conflictDetector.checkForConflicts(config);
 
-      if (results.different.size > 0) {
+      return this.handleConflicts(
+        inputs.data,
+        usernameOrAlias,
+        defaultPackageDir,
+        results
+      );
+    }
+    return { type: 'CANCEL' };
+  }
+
+  public async handleConflicts(
+    manifest: string,
+    usernameOrAlias: string,
+    defaultPackageDir: string,
+    results: DirectoryDiffResults
+  ): Promise<ContinueResponse<string> | CancelResponse> {
+    const conflictTitle = nls.localize(
+      'conflict_detect_view_root',
+      usernameOrAlias,
+      results.different.size
+    );
+
+    if (results.different.size === 0) {
+      conflictView.visualizeDifferences(conflictTitle, usernameOrAlias, false);
+    } else {
+      channelService.appendLine(
+        nls.localize(
+          'conflict_detect_conflict_header',
+          results.different.size,
+          results.scannedRemote,
+          results.scannedLocal
+        )
+      );
+      results.different.forEach(file => {
+        channelService.appendLine(join(defaultPackageDir, file));
+      });
+      channelService.showChannelOutput();
+
+      const choice = await notificationService.showWarningModal(
+        nls.localize(this.messages.warningMessageKey),
+        nls.localize('conflict_detect_override'),
+        nls.localize('conflict_detect_show_conflicts')
+      );
+
+      if (choice !== nls.localize('conflict_detect_override')) {
```

Comment:
If we choose to override the conflicts, do we want to clear the conflict view? Maybe a question for @smaddox-sf

Reply:
Yes, this was an oversight, the view should be cleared.